etc/spack/defaults/packages.yaml

```diff
@@ -14,10 +14,16 @@
 #   ~/.spack/packages.yaml
 # -------------------------------------------------------------------------
 packages:
+  ibm-mpi:
```

Comment:
I understand this is a temporary patch for us to build some packages with spectrum_mpi. Then, what would be the best way to build packages with spectrum_mpi?  Should I generate another packages.yaml in .spack?